builtin/credential/github/path_config.go

```diff
@@ -18,21 +18,25 @@ func pathConfig(b *backend) *framework.Path {
 			"organization": &framework.FieldSchema{
 				Type:        framework.TypeString,
 				Description: "The organization users must be part of",
+				DisplayName: "Organization",
```

Comment:
What sort of logic can we rely on the UI performing to turn a field name into a displayed name? I think I'd like it if DisplayName definition would be an exception and not generally required. We can make new fields conform to what the UI will infer to help that along. In this case, are we not automatically just title-casing the name if DisplayName isn't present? The other common cases would be: 1. splitting on underscore into title-cased words 2. substitution of very common acronyms (url, ttl, mfa, arn, etc.).

I definitely don't want the JS bloated with a bunch of special cases (e.g. "org"->"organization" as a DisplayName is totally fine), but I'm thinking how we can best get most of the correct names directly from the property name.

cc: @meirish

Reply:
This makes sense. I'll go through and clean up.